### PR TITLE
DD: Fix gcc 8 Warning

### PR DESCRIPTION
**Pull request by ianna**
* No need for a try block

### CODE REVIEW

**Comment by ianna** [comment]
please test

**Comment by ianna** [comment]
+1

File: DetectorDescription/RegressionTest/src/DDErrorDetection.cc [hunk 1]
```diff
@@ -221,24 +221,18 @@ const std::map<DDSolid,std::set<DDSolid> > & DDErrorDetection::so()
...
-    catch(DDSolid m) {
-      ;
-      //std::cout << "no such material: " << m << " for creating a walker." << std::endl;
+    ma_walker_t w(mag,*mit);
+    while (w.next()) {
+      result_[*mit].insert(w.current().first);
     }
-   } 
-   return result_;
+    std::cout << std::endl;
```
Comment (fabiocos) [inline DetectorDescription/RegressionTest/src/DDErrorDetection.cc]: minor thing: what's the purpose of this line?

Comment (ianna) [reply DetectorDescription/RegressionTest/src/DDErrorDetection.cc]: it's there to have an extra line for a nicer output.

**Comment by fabiocos** [comment]
+1

**Pull request merged**